piccolo3/server/PiccoloCoolboxControl.py

```diff
@@ -35,14 +35,20 @@
 class PiccoloSerialConnection(PiccoloBaseComponent):
     """Manage serial connection for controlling and managing the coolbox"""
 
-    def __init__(self, serial_port="/dev/ttyUSB0"):
+    async def __init__(self, serial_port="/dev/ttyUSB0"):
```

Comment:
Hi @mhagdorn, I'm having trouble finding a way around this. This as it is currently written can't be done (i.e. async/await-ing init), as it means init returns a coroutine, which isn't allowed. However, the setup is async. Any smart ideas, or am I missing something obvious?

Reply:
see my comment above. you need to create a task and schedule it in the __init__

Reply:
Ah I missed that one 🤦 apologies!

Reply:
no worries, I learned that you can't have async __init__

Reply:
That does seem to have fixed it 🤞! I'll let you know after more testing...

Reply:
Similar to the above comment, I think this worked too.